cellarium/cas/__init__.py

```diff
@@ -1,4 +1,15 @@
-from cellarium.cas.client import CASClient  # noqa
-from cellarium.cas.version import get_version  # noqa
+from . import constants, exceptions, postprocessing, preprocessing, service, settings, version, visualization
```

Comment:
Note that by importing the visualisation in the root package here, we no longer can _not_ install with the `vis` requirements.  Are we ok with that?